packages/color-area/test/color-area.test.ts

```diff
@@ -217,11 +222,10 @@ describe('ColorArea', () => {
         await el.updateComplete;
 
         const handle = el.shadowRoot.querySelector('.handle') as ColorHandle;
-
-        expect(handle.color).to.equal('hsl(0, 100%, 50%)');
+        expect(handle.color).to.equal('hsl(0 100% 50%)');
         el.y = 0.7;
         await el.updateComplete;
-        expect(handle.color).to.equal('hsl(0, 100%, 35%)');
+        expect(handle.color).to.equal('hsl(0 100% 35%)');
```

Comment:
Nope, our color controller accepts colors in a flexible formats but it outputs them in a specific std. format, either in a CSS string (using color.js native `toString()` method which converts native color objects into css strings) or in object format. 
Since sp-color-handle does not leverage any color library and merely requires a css color string to assign itself a color value, thus sp-color-area assigns a css color string to sp-color-handle by leveraging the `toString()` method of color.js which returns colors in space separated format.
For more info:[](https://colorjs.io/docs/output)